x-pack/test/apm_api_integration/tests/services/throughput.ts

```diff
@@ -143,12 +144,14 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     'Throughput when data is loaded with time comparison',
     { config: 'basic', archives: [archiveName] },
```

Comment:
What about moving this section into the above registry section since they use the same archive